Jenkinsfile_CNP

```diff
@@ -40,35 +39,6 @@ withPipeline(type, product, component) {
       env.CORE_CASE_DATA_API_URL = 'http://ccd-data-store-api-aat.service.core-compute-aat.internal'
       env.IDAM_API_URL = 'https://preprod-idamapi.reform.hmcts.net:3511'
       env.IDAM_CLIENT_REDIRECT_URI = 'https://rpe-bulk-scan-processor-sandbox.service.core-compute-sandbox.internal/oauth2/callback'
-
-      def az = { cmd -> return sh(script: "env AZURE_CONFIG_DIR=/opt/jenkins/.azure-${env.SUBSCRIPTION_NAME} az $cmd", returnStdout: true).trim() }
-
-      def dockerImage = new DockerImage(product, component, null, env.BRANCH_NAME)
-      def subscription = env.SUBSCRIPTION_NAME
-      def aksServiceName = dockerImage.getAksServiceName().toLowerCase()
-      def serviceBusSecret = "${aksServiceName}-servicebus-secret"
-      def queueNames = [
-        "envelopes",
-        "notifications"
-      ]
-      def namespace = aksServiceName
-      def kubectl = new Kubectl(this, subscription, namespace)
-      kubectl.login()
-
-      // Get service bus connection string and parse to get namespace
-      def serviceBusConnectionStr = kubectl.getSecret(serviceBusSecret, namespace, "{.data.connectionString}")
-      //Regex: Endpoint=sb:\/\/([-a-zA-Z0-9]+)\.servicebus\.windows\.net.*
-      def serviceBusNamespace = serviceBusConnectionStr.substring(14).split("\\.")[0]
-      env.QUEUE_READ_CONN_STRING = "${serviceBusConnectionStr};EntityPath=envelopes"
-      env.QUEUE_WRITE_CONN_STRING = "${serviceBusConnectionStr};EntityPath=envelopes"
```

Comment:
I think you still need this, otherwise how is the functional test getting its connection string?